### PR TITLE
[DOCS] Provide segment about Header/FooterAssets

### PR DESCRIPTION
**Pull request by helsner**

### CODE REVIEW

**Comment by DavidBruchmann** [comment]
Great, what you think about some example code?

**Comment by helsner** [comment]
I could provide that if you want to, but i could not really see the big advantage of that IMHO as the info is pretty straight forward. Maybe for visual separation reasons
But i would add that on your call!

**Comment by DavidBruchmann** [comment]
The info in the Changelog is more extensive than your description (also including the option to override `renderAssetsForRequest`).
So it would be good if the same info is at east available - this can be copy & paste fully or partially, that's no problem.  
An example would be nice I think, but it could be rudimentarily, just to show the general usage.

File: Documentation/typo3fluid/fluid/latest/Section.rst [hunk 1]
```diff
@@ -61,6 +61,14 @@ Special sections HeaderAssets/FooterAssets
 In case you want to add header or footer assets within plugins, you can use the special sections "HeaderAssets" and "FooterAssets".
 Those are rendered by the ActionController (\TYPO3\CMS\Extbase\Mvc\Controller\ActionController) and provide the possibilty to add resources, meta data etc. as you like by simply using those two sections as wanted.
 
+::
+
+    <f:section name="HeaderAssets">
+       <title>My title within the plugin template</title>
```
Comment (DavidBruchmann) [inline Documentation/typo3fluid/fluid/latest/Section.rst:67]: <link rel="stylesheet" href="typo3conf/ext/my_extension/Resources/Public/Css/myCssFile.css" />

Comment (DavidBruchmann) [reply Documentation/typo3fluid/fluid/latest/Section.rst:67]: the title-tag might be shown twice (with different content), if not the whole header-area is rendered individually.

File: Documentation/typo3fluid/fluid/latest/Section.rst [hunk 2]
```diff
@@ -61,6 +61,14 @@ Special sections HeaderAssets/FooterAssets
...
+    </f:section>
+    <f:section name="FooterAssets">
+         <p>© My example copyright note in the footer</p>
```
Comment (DavidBruchmann) [inline Documentation/typo3fluid/fluid/latest/Section.rst:70]: `<script src="typo3conf/myExtension/my_extension/Resources/Public/Js/myJsFile.js"></script>`

File: Documentation/typo3fluid/fluid/latest/Section.rst [hunk 3]
```diff
@@ -61,6 +61,14 @@ Special sections HeaderAssets/FooterAssets
 In case you want to add header or footer assets within plugins, you can use the special sections "HeaderAssets" and "FooterAssets".
 Those are rendered by the ActionController (\TYPO3\CMS\Extbase\Mvc\Controller\ActionController) and provide the possibilty to add resources, meta data etc. as you like by simply using those two sections as wanted.
```
Comment (DavidBruchmann) [inline Documentation/typo3fluid/fluid/latest/Section.rst:63]: When rendering, `{request}` is available as template variable in both sections, as is `{arguments}`, allowing you to make decisions based on various request/controller arguments. As usual, `{settings}` is also available.

All content you write into these sections will be output in the respective location as is, meaning you must write the entire `<script>` or whichever tag you are writing, including all attributes. You can of course use various Fluid ViewHelpers to resolve extension asset paths.

The feature only applies to ActionController (thus excluding CommandController) and will only attempt to render the section if the view is an instance of `TYPO3Fluid\\Fluid\\View\\TemplateView` (thus including any View in TYPO3 which extends either TemplateView or AbstractTemplateView from TYPO3’s Fluid adapter).

**Comment by DavidBruchmann** [comment]
@helsner I hope the backticks render it correctly, in the preview above it seems being wrong.

**Comment by helsner** [comment]
Looks good to me! I mind the edits, as they mean i could've done better 😝 
Thanks for your help and exchange on this topic

**Review by tomwarwick — APPROVED** [review]

**Comment by DavidBruchmann** [comment]
@helsner you found it and cared about it, that's most important ;-)

**Comment by linawolf** [comment]
@helsner I fear susannemoog is right. Unfortunarely you wouldhave to move this to the source code. 

I put your PR on draft so none merges it accidently.

**Comment by helsner** [comment]
Hi y'all - could you help me out where i find the correct source? just checked the core without success 😬

**Comment by susannemoog** [comment]
@helsner sure: https://github.com/TYPO3/Fluid/blob/master/src/ViewHelpers/SectionViewHelper.php#L17 - the hint to find it is the package name in the file path (typo3fluid/fluid === TYPO3/Fluid, so the main fluid repository)

**Comment by helsner** [comment]
Thx for the hint - then i was right already.. i wasn't sure.
https://github.com/TYPO3/Fluid/pull/558

**Comment by linawolf** [comment]
was solved with https://github.com/TYPO3/Fluid/pull/558

**Pull request closed without merging**